openstack/compute/v2/flavors/requests.go

```diff
@@ -21,22 +21,12 @@ type ListOptsBuilder interface {
 //
 // If no IsPublic argument is supplied in the request, OpenStack treats the FlavorType as Project.
 type FlavorType string
+
 const (
-        Project FlavorType =  "true"
-        Private FlavorType = "false"
-        All FlavorType = "None"
+	Project FlavorType = "true"
```

Comment:
You making that comment made me realize you're correct, but I also think `Project` muddies things, too.

What about `FlavorScope` or `FlavorAccessScope`?